openpype/plugins/publish/extract_slate_global.py

```diff
@@ -0,0 +1,541 @@
+import os
```

Comment:
This plugin is using python modules that may cause crashes in some DCCs. Maybe some of the logic should be separated to a OpenPype script.

Reply:
> If the latter, the plugin wouldn't show up on the DCC but that's it? I guess you are referring to html2image only?

Yes and yes.

Reply:
so it won't "crash" and it can be ignored?

Although as long as we add html2image as a dependency on poetry it won't error out just by importing it like any other dependencies. Not sure I follow on the advantage of separating into a script